__mocks__/@nebula.js/stardust.js

```diff
@@ -1,8 +1,32 @@
 const { useState, useMemo, useEffect } = require("react");
 
-const usePromise = (callback) => {
-  callback();
-  return [Promise.resolve(), undefined];
+const usePromise = (callback, deps) => {
```

Comment:
why we are mocking this one? 
it's source exactly doing the same [but in useLayoutEffect](https://github.com/qlik-oss/nebula.js/blob/91e04c9a0e1df94cf17edeed6dd9603e7b6de03e/apis/supernova/src/hooks.js#L411-L456)

Reply:
The original version can only be run in supernova context. So we need to mock it.